agent/src/index.ts

```diff
@@ -335,6 +335,7 @@ export async function initializeClients(
     }
 
     if (clientTypes.includes("twitter")) {
+        TwitterClientInterface.enableSearch = getSecret(character, "TWITTER_SEARCH_ENABLE");
```

Comment:
Can we add this to the .env.example? TWITTER_SEARCH_ENABLE

Reply:
Added